packages/versioning/test/incompatible-versioning.test.ts

```diff
@@ -103,51 +104,43 @@ describe("versioning: validate incompatible references", () => {
   });
 
   describe("operation", () => {
-    it("emit diagnostic when unversioned op has a versioned model as a parameter", async () => {
-      const diagnostics = await runner.diagnose(`
+    it("allow unversioned op to have versioned model as a parameter", async () => {
```

Comment:
I think the test before was correct, if model `Foo` was added in v2 there shoulnd't be a way to use in in v1